### ISSUE TITLE
Flashing the Arduino Code - enhancement to avoid file name conflict and confusion

### ISSUE THREAD

**Issue by JHPHELAN** [nasa-jpl/osr-rover-code#115]
In the instructions for Arduino setup:
 [https://github.com/nasa-jpl/osr-rover-code/blob/master/setup/arduino.md](url)
it suggests to download the repo from here
 [https://github.com/nasa-jpl/osr-rover-code](url)
zip the Arduino folder and rename it OsrScreen.zip.
However, when the Arduino IDE unzips it, it still files it as its original name "Arduino" in the IDE library here:
C:\Users\Me\Documents\Arduino\libraries
There are many files already named Arduino-something in that library.
It would be less confusing, and cause less potential conflict in the IDE library directory to change the Arduino folder to OsrScreen in the repo.  When it is zipped & reopened by the IDE, the name will be OsrScreen instead of the more ambiguous Arduino.

The install instructions would also benefit from this additional information:  If the repo is re-downloaded for any reason, and you attempt to reinstall the Arduino/OsrScreen library, you will get the error message "A library named Arduino/OsrScreen already exits."  In which case you need to go to the C:\Users\Me\Documents\Arduino\libraries directory, delete the old file then reinstall the library.

**Comment by Achllle** [comment]
This would be a great candidate for you to try to [contribute yourself](https://github.com/firstcontributions/first-contributions). For the first part, you'd simple rename the directory, commit your changes, and open a PR (pull request).

The second part seems rather obvious and would be something that people could ask on Github Discussions, Slack, or Tapatalk.

Finally, please take a moment to familiarize yourself with [Markdown](https://guides.github.com/features/mastering-markdown/). It's really hard to parse through your text.
